pegasus/sites.v3/code.org/public/teacher-dashboard/section_manage.haml

```diff
@@ -172,3 +172,9 @@ content-type: text/ng-template
 
   #login-type-react{'ng-init' => '$emit("login-type-react-rendered");'}
   #share-setting-react{'ng-init' => '$emit("share-setting-react-rendered");'}
+
+  %br/
+  %div{id: 'privacy_link'}
+    %a{'ng-href' => CDO.code_org_url("/privacy/student-privacy"), target: "_blank"}
+      =I18n.t('click_here')
+    =I18n.t('privacy_doc_explanation')
```

Comment:
Technically this is much better/safer. However, I would check with PM, as I'm not sure this structure will translate well (i.e. when translating `privacy_doc_explanation` I don think translators necessarily get the context, which is tough for a sentence fragment like this.

I wonder if we should just have the entire sentence be a link.